test/prism/fixtures/whitequark/it_arg_after_34.txt

```diff
@@ -0,0 +1,7 @@
+-> do it + 42 end
```

Comment:
This code is actually not really tested, only `Parser33` is run in the test suite. I thought about it a bit before, and probably a separate directory for new syntax would make sense (or something along those lines).

You probably need to assert specifically against the ast anyways, as it only compares to the parser gem, and there is nothing to compare against in this case.

Reply:
For now, I've added a test to check the `:itblock` node type, which is currently provided only by `Prism::Translation::Parser`. Since it is essentially an extension of the Parser gem, I've added it to `parser_test.rb`.  

Additionally, I've implemented a mechanism to compare node compatibility with the `Parser::Ruby34`, but since the Parser gem doesn't yet support the `it` block parameter, it is currently skipped.